content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started-mblock/getting-started-mblock.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Start With Alvik And Block-Based Language"
+difficulty: beginner
+description: "Take your first steps with Arduino® Alvik and mBlock."
+tags:
+  - Robot, Block-based, Education
+author: "Gaia Castronovo"
+---
+
+![Alvik Robot](assets/getting-started.jpg)
+
+Welcome to your first steps with Alvik, your companion for exploring robotics and programming. In this guide, we'll walk you through setting up your Alvik and starting your journey with block based language.
+
+## What’s Included in the Box 
+
+When unboxing Alvik, you should find the following items:
+
+- Alvik robot
+- USB-C® cable (located under the cardboard tray)
+
+## Update Firmware with Alvik Updater
+
+Updating the firmware on your Alvik ensures it has the latest features, bug fixes, and performance improvements. Regular updates help maintain compatibility with the mBlock software and ensure optimal performance of your robot.
+
+![Alvik Firmware Updater](assets/AlvikFirmwareUpdater30sec.gif)
+
+1. Visit https://alvikupdate.arduino.cc.
+2. Connect Alvik to your computer.
+3. Turn ON your Alvik.
+4. Click the "Connect" button.
+5. **A pop-up window will appear** prompting you to select the COM port. Choose the correct port and confirm.
+6. Once connected, click the "Update" button and wait for the process to complete.
+
+## Preparing Your Environment
+
+During the installation process, you may **need to install additional drivers**, that require access to the **administrator login password** to your OS.
+
+### Supported Systems
+
+You can use a computer with different operating systems like:
+
+* Windows
+* Mac, 
```

Comment:
macOS is the OS for Mac. This is a small and not important detail perhaps but it was what we have been used in some cases like https://docs.arduino.cc/tutorials/alvik/user-manual/